games-puzzle/tetzle/tetzle-2.2.3.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="Tetromino jigsaw puzzle game"
+DESCRIPTION="Tetzle is a jigsaw puzzle game that uses tetrominoes \
+for the pieces. Any image can be imported and used to create \
+puzzles with a wide range of sizes. Games are saved \
+automatically, and you can select between currently in progress \
+games."
+HOMEPAGE="https://gottcode.org/tetzle/"
+COPYRIGHT="2008-2024 Graeme Gott"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://gottcode.org/tetzle/tetzle-$portVersion.tar.bz2"
+CHECKSUM_SHA256="e70b15eb736d14f8aee60e0618599973c6b1beac22e8ee996bb880f332265532"
+SOURCE_DIR="tetzle-$portVersion"
+ADDITIONAL_FILES="tetzle.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	tetzle$secondaryArchSuffix = $portVersion
+	app:Tetzle$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libQt6Core$secondaryArchSuffix
+	lib:libQt6Gui$secondaryArchSuffix
+	lib:libQt6Widgets$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	qt6_tools${secondaryArchSuffix}_devel
```

Comment:
Did you try using `devel:libQt6UiTools$secondaryArchSuffix` instead of this?

Reply:
Not yet, as I did this upon the advice of @augiedoggie. I'll try it with another recipe in progress.